src/pages/search/styles.module.css

```diff
@@ -22,3 +22,23 @@
   color: var(--primary-color);
   text-decoration: underline;
 }
+
+.menu{
+  z-index: 1;
+  width: 80%;
```

Comment:
i think we should remove the .search class and the associated div, and just add `max-width: 720px;` to this class. we can also remove the `@media` query

Reply:
Ok, removed surrounding div,  .search class, and @media.search class. 
Added max-width to .menu class. 
I also added width:100% in SearchBar component so as to fill entire space of container here. And removed media query in SearchBar since it became redundant.